INFO     root:test_case_excel.py:28 [{'time': '2024-01-11 14:46:03|48\n'}]
INFO     root:test_case_excel.py:30 从excel获取的测试用例数据！！！！
INFO     root:test_case_excel.py:31 (13, '协商成交（XSCJ_SHG）_反向用例_交易价格位数校验', '上交所协商成交_交易价格校验', '上交所协商成交_交易价格校验_四位小数，最后一位不为0', 'blocker', 1101, 'data/shg_fix/DS_1101_SHG_Fix_XSCJ.xml', '{"matchNo":"122","sysOrdID":"","dir":"S","bOrdCount":"0","bOrdPrice":"0","sOrdCount":"100","sOrdPrice":"102.8888"}', 'select t.ordstatus,t.errinfo,t.errcode from ttrd_fix_order t  where t.sysordid={0}', 'all_val["sysOrdID"]', '{"ORDSTATUS":2,"ERRCODE":-2008,"ERRINFO":":Tag=44:7004|小数点后保留位数错误"}', '测试通过', 'yes', None, 'data/step_case_shg.xlsx', '上交所固收（XSHG_FIX）')
INFO     root:deal_xml.py:27 组装报文完成！
INFO     root:deal_xml.py:42 读取XML完成！
INFO     root:test_case_excel.py:88 <?xml version='1.0' encoding='utf-8'?>
<root>
  <header>
    <msgType>R</msgType>
    <action>1101</action>
    <clientID>xIR</clientID>
    <reqSn>7897b7e19186472f823bad3d04ec9799</reqSn>
    <sendDateTime>2024-01-11 14:51:08</sendDateTime>
    <fromIPMAC>191.168.0.53,</fromIPMAC>
  </header>
  <body>
    <orders>
      <order>
        <secuExtAccid>B883056635</secuExtAccid>
        <orderType>9</orderType>
        <dir>S</dir>
        <iCode>010609</iCode>
        <sysOrdID>1704955868</sysOrdID>
        <iName>MDe5+tWuMDY=</iName>
        <exeMarket>XSHG_FIX</exeMarket>
        <ownerType>103</ownerType>
        <bOrdCount>0</bOrdCount>
        <bOrdPrice>0</bOrdPrice>
        <sOrdCount>100</sOrdCount>
        <sOrdPrice>102.8888</sOrdPrice>
        <settlType>103</settlType>
        <settlPeriod />
        <lastParPx>0</lastParPx>
        <cashMargin>1</cashMargin>
        <memo />
        <ordDate>2024-01-11</ordDate>
        <ordTime>14:51:08</ordTime>
        <matchNo>122</matchNo>
        <product>01</product>
        <tradeType>10</tradeType>
        <company>316</company>
        <trader>Z31608</trader>
        <seatno>88045</seatno>
        <obTrader>Z31606</obTrader>
        <iceCount>0</iceCount>
        <clientOrdid>lwy111</clientOrdid>
        <accname>lwy_shg_B883056635</accname>
        <shareProperty>01</shareProperty>
      </order>
    </orders>
    <accountInfo>
      <cashExtAccID>lwy_wbzj_jys</cashExtAccID>
      <cashExtAccName>lwy_shg_001</cashExtAccName>
      <cashAccID>LWY_NBZJ_JYS</cashAccID>
      <cashAccIDAccName>LWY_NBZJ_JYS</cashAccIDAccName>
      <xirSecuExtAccID>lwy_wbzq_sh_02</xirSecuExtAccID>
      <xirSecuExtAccName>lwy_wbzq_sh_02</xirSecuExtAccName>
      <secuAccID>LWY_NBZQ_JYS</secuAccID>
      <secuAccName>LWY_NBZQ_JYS</secuAccName>
    </accountInfo>
    <terminalInfo>
      <clientPublicIpAddress />
      <clientPublicPort />
      <clientInnerIpAddress />
      <clientMacaddress />
      <clientHarddiskno />
      <clientPcname />
      <clientCpuno />
      <clientDiskpartitioninfo />
      <clientReellabel />
      <clientName>xIRCS</clientName>
      <clientVersion />
      <isTerminalinfo>0</isTerminalinfo>
    </terminalInfo>
  </body>
</root>
INFO     root:tools.py:24 发送POST请求完成
INFO     root:test_case_excel.py:91 (<HTTPStatus.OK: 200>, <?xml version="1.0" encoding="UTF-8" ?>
<root>
  <header>
    <msgType>A</msgType>
    <action>1101</action>
    <clientID>xIR</clientID>
    <reqSn>7897b7e19186472f823bad3d04ec9799</reqSn>
    <sendDateTime>2024-01-11 14:51:08</sendDateTime>
    <fromIPMAC>191.168.0.53,</fromIPMAC>
    <errorCode>0</errorCode>
    <errorInfo></errorInfo>
    <massageIndex>0</massageIndex>
  </header>
  <body>
    <reqID>0</reqID>
    <reqRefID>0</reqRefID>
    <extReqID></extReqID>
    <extReqRefID></extReqRefID>
    <reqStatus>0</reqStatus>
    <errorCode>0</errorCode>
    <errorInfo></errorInfo>
  </body>
</root>)
ERROR    root:run_shg_task.py:21 任务执行成功!
INFO     root:test_case_excel.py:135 select t.ordstatus,t.errinfo,t.errcode from ttrd_fix_order t  where t.sysordid=1704955868
INFO     root:test_case_excel.py:143 {'ORDSTATUS': 2, 'ERRINFO': ':Tag=44:7004|小数点后保留位数错误', 'ERRCODE': -2008}
INFO     root:test_case_excel.py:148 测试通过
INFO     root:test_case_excel.py:148 测试通过
INFO     root:test_case_excel.py:148 测试通过